go/sentry/api/api.go

```diff
@@ -13,9 +14,22 @@ type SentryAddresses struct {
 	Committee []node.CommitteeAddress
 }
 
+// UpstreamTLSCertificates contains the upstream TLS certificates.
+type UpstreamTLSCertificates struct {
+	Certificate     *tls.Certificate
```

Comment:
This is serializable so add explicit field annotations. Also it should probaby not use `tls.Certificate` directly but only be a `[]byte`.

Alternative: just use a single argument of type `[][]byte` to include all certificates.

Reply:
Thanks for the suggestion, I've switched to using `[][]byte` for all these cases.